default/stringtables/en.txt

```diff
@@ -4355,7 +4355,7 @@ RESEARCH_TEXT
 '''The Research (or RP "Research Points") of all planets in the empire is combined to discover new technologies. Any Research that cannot be spent on a tech is lost each turn. Each tech has a minimum number of turns required to research it. For example, if a tech that cost 15 RPs, and had a minimum time of 3 turns, 5 RPs is the maximum that could be put towards its completion per turn.
 The tech at the top of the queue is given RPs first. Any remaining RP are then given to the next tech in the queue, and if there is still more remaining to the next-- and so on. You can re-prioritize the items in the queue by dragging the most important items closer to the top.
 
-Research does not require [[encyclopedia SUPPLY_TITLE]] line connections to be used.'''
+Research does not require [[encyclopedia SUPPLY_TITLE]] line connections to be used. However, if a tech which improves a ship part is researched (e.g. a weapon), the already existing ships must be in supply range, stationary, and not in combat, in order to receive the upgrade during the following turn.'''
```

Comment:
The research article is linked from ~30 stringtable entries.  If not dropped, I'd opt for a new paragraph here with a more general wording (ship parts and supply lines are not the only requirement pairs).

Something like:

`Research that unlocks an item will only give an empire the knowledge of how to construct that item.  Actually constructing such an item may have other requirements, such as access to a certain resource.`
`If an item has been unlocked and is not available, check the items pedia entry for requirements.`
`Another option is to check the hover text on the items listing in the production window (with Unavailable selected).`